File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    org=x;
    if(org<0)
    return false;
    var rev=0;
    var r;
    while(x>0){
        r=x%10;
        rev=rev*10+r;
        x=Math.floor(x/10);
    }
    if(rev==org)
    return true;
    else
    return false
};

File: 0009-palindrome-number/0009-palindrome-number.c
bool isPalindrome(int x) {
    int temp=x;
     long int r,rev=0,rem;
    while(x>0)
    {
        r=x%10;
        rev=rev*10+r;
        x=x/10;
    }
    if(rev==temp)
    return true;
    else
    return false;
}